src/sqlite/sqlitedatabase.android.ts

```diff
@@ -146,7 +146,7 @@ const transactionRaw = async <T = any>(db: Db, action: (cancel?: () => void) =>
     }
 };
 
-const messagePromises: { [key: string]: { resolve: Function; reject: Function; timeoutTimer: NodeJS.Timer }[] } = {};
+const messagePromises: { [key: string]: { resolve: Function; reject: Function; timeoutTimer: ReturnType<typeof setTimeout> }[] } = {};
```

Comment:
Had to make this small change otherwise it wouldn't compile. {N} typings use `number` for setTimeout instead of `NodeJS.Timer`.